Server/src/RedisManager.cpp

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (c) 2024 Microsoft, Inc. and others.  All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ */
+
+#include "RedisManager.h"
+
+
+/*********************************************************************//**
+ * Constructor for class
+ ***********************************************************************/
+RedisManager::RedisManager() : stateDb_(BMP_DB_NAME, 0, true) {
+    swss::SonicDBConfig::initialize();
+    swss::SonicDBConfig::initializeGlobalConfig();
```

Comment:
These 2 line are conflict, initialize if for single asic device, initializeGlobalConfig is for multiasic device.

Not sure if this service need handle multiple asic case. if not, use swss::SonicDBConfig::initialize() is enough.

But if it needs, suggest adding an app parameter and initialize SonicDBConfig like this python code:
    if is_multi_asic():
        if not SonicDBConfig.isGlobalInit():
            SonicDBConfig.initializeGlobalConfig()
    else:
        if not SonicDBConfig.isInit():
            SonicDBConfig.initialize()

Reply:
Added one config option for this.